graphql_compiler/cost_estimation/cardinality_estimator.py

```diff
@@ -227,7 +228,8 @@ def estimate_query_result_cardinality(
         lookup_class_counts: function, string -> int, that accepts a class name and returns the
                              total number of instances plus subclass instances
         graphql_query: string, a valid GraphQL query
-        parameters: dict, parameters with which query will be executed
+        parameters: dict, parameters with which query will be executed. Must be valid for the
+                    given graphql_query.
```

Comment:
I think this doesn't communicate anything. Of course, the parameters must be valid.